ts/features/wallet/onboarding/cobadge/screens/start/CoBadgeStartScreen.tsx

```diff
@@ -39,17 +48,35 @@ const CoBadgeStartScreen = (props: Props): React.ReactElement => {
   }
   switch (props.abiSelectedConfiguration.value) {
     case StatusEnum.enabled:
+      void trackOutput("Enabled", props.maybeAbiSelected);
       // Single ABI (bank) screen that allow to start the search
       return <CoBadgeChosenBankScreen abi={props.maybeAbiSelected} />;
     case StatusEnum.disabled:
+      void trackOutput("Disabled", props.maybeAbiSelected);
       // The chosen ABI is disabled (not yet available)
       return <CoBadgeStartKoDisabled />;
     case StatusEnum.unavailable:
+      void trackOutput("Unavailable", props.maybeAbiSelected);
       // THe chosen ABI is unavailable (technical problems)
       return <CoBadgeStartKoUnavailable />;
   }
 };
 
+const CoBadgeStartMemo = React.memo(
+  CoBadgeStartComponent,
+  (prev: Props, curr: Props) =>
+    prev.maybeAbiSelected === curr.maybeAbiSelected &&
+    prev.abiSelectedConfiguration.kind === curr.abiSelectedConfiguration.kind &&
```

Comment:
Maybe these conditions

```typescript
pot.isSome(prev.abiSelectedConfiguration) &&
pot.isSome(curr.abiSelectedConfiguration) &&
```

could be removed?

I guess if this is true
`prev.abiSelectedConfiguration.kind === curr.abiSelectedConfiguration.kind`
also the previous condition will be. Isn't right?